lms/envs/common.py

```diff
@@ -485,6 +485,10 @@
 # e.g. COURSE_BLOCKS_API_EXTRA_FIELDS = [  ('course', 'other_course_settings'), ("problem", "weight")  ]
 COURSE_BLOCKS_API_EXTRA_FIELDS = []
 
+# Setting to redirect unauthenticated users to logistration page when enrolling from course about
+# page instead of showing error message. After successful login redirect them back to the course about page.
```

Comment:
```suggestion
# Setting to redirect unauthenticated users to the login page when enrolling from the course about
# page instead of showing an error message. After successful login redirects them back to the course about page.
```